roda-ui/roda-wui/src/main/java/org/roda/wui/client/common/slider/InfoSliderHelper.java

```diff
@@ -616,6 +616,24 @@ private static FlowPanel createRepresentationTypeHTML(BrowseRepresentationBundle
     return panel;
   }
 
+  public static void addLinkifCentralInstance(HashMap<String, Widget> values, String instancename, String instanceId) {
+    if (StringUtils.isNotBlank(instanceId)) {
+      String distributedMode = ConfigurationManager.getStringWithDefault(
+        RodaConstants.DEFAULT_DISTRIBUTED_MODE_TYPE.name(), RodaConstants.DISTRIBUTED_MODE_TYPE_PROPERTY);
+      if (distributedMode.equals(RodaConstants.DistributedModeType.CENTRAL.name())) {
+        Anchor anchor = new Anchor();
+        if (StringUtils.isNotBlank(instancename)) {
+          anchor.setText(instancename);
+        } else {
+          anchor.setText(instanceId);
+        }
+        anchor.setHref(HistoryUtils.createHistoryHashLink(ShowDistributedInstance.RESOLVER, instanceId));
+        values.put(messages.distributedInstanceLabel(), anchor);
+      } else {
+        values.put(messages.itemInstanceId(), new Label(instanceId));
+      }
+    }
+  }
   private static FlowPanel createAipInstanceIdHTML(BrowseAIPBundle bundle) {
```

Comment:
Make sure this is not being used anymore, if not, please delete it. 
Also check `createRepresentationInstanceIdHTML` and `createFileInstanceIdHTML`

Reply:
Done